assets/calc/wrong-type.yml

```diff
@@ -0,0 +1,7 @@
+---
+meta:
+  string: "Hello, World!"
+
+jobs:
+- name: one
+  instances: (( calc "3.1415 * meta.string" ))
```

Comment:
Thanks! That actually revealed a panic. I will fix it and update the pull request.